02-peer-protocol.md

```diff
@@ -856,6 +868,14 @@ seconds, and the protocol only supports an expiry in blocks.
 specification; larger amounts are not necessary, nor wise, during the
 bootstrap phase of the network.
 
+The node _responsible_ for paying the Bitcoin fee should maintain a "fee
+spike buffer" on top of its reserve to accommodate a future fee increase.
+Without this buffer, the node _responsible_ for paying the Bitcoin fee may
+reach a state where it is unable to receive any non-dust HTLC while maintaining
```

Comment:
unable to _send or receive_ ?

Reply:
See this long (but interesting) discussion around that: https://github.com/lightningnetwork/lightning-rfc/pull/740#discussion_r399197774 :)

TL;DR: we probably want to prioritize receiving here. If you still feel it's better to mention both, let me know.

Reply:
If you are describing the limitations of the channel state in this paragraph, there is nothing to prioritize? But I don't mind just mentioning "receive" here. The statement isn't incorrect.

Reply:
I think that what's slightly confusing with sending here is that you also need to reserve the amount you want to send. But it's quite a nit IMO, I could definitely go with `send nor receive` if others also lean towards that.

Reply:
I also tend to `send nor receive any non-dust HTLC`

Reply:
Alright let's go for send nor receive then.

Reply:
Still want to do `send nor receive`? Because that isn't in the current version of the pr

Reply:
Good catch, I had updated it in the requirements but not here. Done in the latest commit.